root/etc/e-smith/events/actions/nethserver-directory-clean-accountsdb

```diff
@@ -26,7 +26,30 @@ use strict;
 use esmith::AccountsDB;
 
 my $adb = esmith::AccountsDB->open() || die ("Could not open AccountsDB");
+my $domain = `hostname -d`;
```

Comment:
We should not trust it any more, because it relies on gethostbyname(3) which is "network aware".

Alternative: parse ``Sys::Hostname`` - [examples](https://github.com/search?q=org%3ANethServer+Sys%3A%3AHostname&type=Code&utf8=%E2%9C%93).